src/adhocracy_frontend/adhocracy_frontend/static/js/mkResources.ts

```diff
@@ -774,11 +772,6 @@ mkNick = (modulePath : string, metaApi : MetaApi.IMetaApi) : string => {
     }
 };
 
-mkSuperTypes = (modulePath : string, metaApi : MetaApi.IMetaApi) : string => {
-    var superTypes : string[] = metaApi.resources[modulePath].super_types;
-    return JSON.stringify(superTypes);
-};
```

Comment:
why did this still exist if the (presumably) only frontend check (see below) didn't need it anyways?

ResourceUtil.ts:65:
> _.includes(resourceMeta.super_types, resourceType);